pkg/cmd/options/options.go

```diff
@@ -0,0 +1,108 @@
+package options
+
+import (
+	"errors"
+	"net/url"
+	"os"
+
+	"github.com/spf13/pflag"
+)
+
+var (
+	// ServiceID is the ID of the service
+	ServiceID = "b431cc67-89fa-4f4a-8b1b-8f8a70c71b5e"
+	// ServiceName is the name of the service
+	ServiceName = "gitea"
+	// SharedPlanID is the ID of the plan
+	SharedPlanID = "3034df5d-c976-4265-87fd-7d3cfbe54a79"
```

Comment:
Let's call the default plan `default` rahter than `shared`. Or maybe have a `private` and a `public` plan which create a private and a public repo.